src/connectedComponents/ToolbarRow.js

```diff
@@ -10,7 +10,7 @@ import { commandsManager, extensionManager } from './../App.js';
 
 import ConnectedCineDialog from './ConnectedCineDialog';
 import ConnectedLayoutButton from './ConnectedLayoutButton';
-import ConnectedPluginSwitch from './ConnectedPluginSwitch.js';
+// import ConnectedPluginSwitch from './ConnectedPluginSwitch.js';
```

Comment:
should you remove these commented out lines or wanna keep it there for future reference?

Reply:
That's cool with me, just checking if it wasn't an omission.